Assets/MixedRealityToolkit.ThirdParty/MRTK-Quest/Scripts/Input/OculusQuestInputManager.cs

```diff
@@ -307,6 +309,21 @@ protected void UpdateHand(OVRHand ovrHand, OVRSkeleton ovrSkeleton, OVRMeshRende
             }
         }
 
+        public void UpdateHandMaterials()
+        {
+            if (rightHandMaterial)
+                Object.Destroy(rightHandMaterial);
+
+            rightHandMaterial = new Material(MRTKOculusConfig.Instance.CustomHandMaterial);
+
+            if (leftHandMaterial)
+                Object.Destroy(leftHandMaterial);
+
+            leftHandMaterial = new Material(MRTKOculusConfig.Instance.CustomHandMaterial);
+
+            RemoveAllHandDevices();
```

Comment:
Instead of calling remove all hand device, you should look at how the material is used in the OculusQuestHand. 
You should create a function there that updates the material on the renderer, and call that here and in the constructor for OculusQuestHand. Destroying the hands is a bit overkill.

Reply:
Created a public function in `OculusQuestHand.cs` script for updating material to the renderer.